packages/components/select/src/useSelect.ts

```diff
@@ -124,11 +124,13 @@ export const useSelect = (props: ISelectProps, emit) => {
   const selectDisabled = computed(() => props.disabled || form?.disabled)
 
   const hasModelValue = computed(() => {
+    const hasEmptyStringOption = optionsArray.value.some(
+      (option) => option.value === ''
+    )
```

Comment:
**Suggestion:** can we make this a computed since we have multiple occurrances.

Reply:
Done.